src/core/interp.c

```diff
@@ -1013,10 +1013,10 @@ void MVM_interp_run(MVMThreadContext *tc, void (*initial_invoke)(MVMThreadContex
             }
             OP(param_op_i):
             {
-                MVMArgInfo param = MVM_args_get_optional_pos_int(tc, &tc->cur_frame->params,
+                MVMint64 param = MVM_args_get_optional_pos_int(tc, &tc->cur_frame->params,
                     GET_UI16(cur_op, 2));
-                if (param.exists) {
-                    GET_REG(cur_op, 0).i64 = param.arg.i64;
+                if (param) {
```

Comment:
if (param) means if (param != 0) making it impossible to pass a perfectly good 0 as positional int parameter.